sql_user_check_sql_server/README.md

```diff
@@ -7,6 +7,7 @@ Install [sqlcmd](https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-se
 ### Run the script
 
 Below command will ask password.
+For trusting server certificate add -c 1. For Microsoft Entra Authentication add -a 1 
```

Comment:
Mention it as entra authentication or Azure AD authentication

Reply:
Done